Update README

ksbeattie opened the pull request:
## Fixes

Adding badge to RTDs latest build, re-working the getting started and a few other small changes.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

ksbeattie committed:
Update README.me

Adding badge to RTDs latest build, re-working the getting started and a few other small changes.

lbianchi-lbl reviewed (dismissed):
LGTM :+1:

dangunter commented:
I think the quickstart should point to full installation instructions, since particularly the pip version won't get the users to a point where most of the models can be run.

dangunter committed:
Merge branch 'main' into readme_update

dangunter commented on README.md:
```diff
@@ -9,8 +9,31 @@ scale-up, operation and troubleshooting of innovative, advanced energy systems.
+[![Documentation Status](https://readthedocs.org/projects/idaes-pse/badge/?version=latest)](https://idaes-pse.readthedocs.io/en/latest/?badge=latest)
 <!-- END Status badges -->
 
+## Quickstart
```
I think the quickstart should point to full installation instructions, since particularly the pip version won't get the users to a point where most of the models can be run. Also, usually a quickstart ends with running something to see that it worked. This is more of a "quick-install".

ksbeattie replied:
Ok, I tried to re-work things to make it clearer.

ksbeattie committed:
Re-working things

Renamed and added to the Getting Started section
Removed the docs and testing sections as those expected source tree to be there (which is no longer expected)
Expanded the Contributing section
Added some cool new badges

dangunter committed:
Merge branch 'main' into readme_update

ksbeattie committed:
Merge branch 'main' into readme_update

dangunter reviewed (approved):
Great

lbianchi-lbl reviewed (approved):
LGTM :+1:

ksbeattie committed:
Merge branch 'main' into readme_update
